service/src/lib.rs

```diff
@@ -235,6 +235,12 @@ pub use secrets::SecretStore;
 ))]
 pub mod aws;
 
+// TODO - remove this when we have a better way to handle this
+// #[cfg(feature = "persist")]
+
+pub mod persist;
+pub use persist::PersistInstance;
```

Comment:
```suggestion
#[cfg(feature = "persist")]
pub mod persist;
#[cfg(feature = "persist")]
pub use persist::PersistInstance;
```

This will need a `persist` feature to be added in `service/Cargo.toml`